src/isNegative.js

```diff
@@ -3,7 +3,7 @@ import { both, gt } from 'ramda';
 import isNumber from './isNumber';
 
 /**
- * Checks if value is a negative `Number` primitive or object.
+ * Checks if value is a negative `Number` primitive or object. Zero is not considered negative.
```

Comment:
Checks if value is a negative `Number` primitive or object. Zero is considered neither positive nor negative.